src/engine/fileOps/tfm.cpp

```diff
@@ -511,6 +516,29 @@ void TFMParsePattern(struct TFMParsePatternInfo info) {
       }
     }
   }
+
+  if (lastPatSeen>1) {
+    // clone the last pattern
+    info.maxPat++;
+    for (int i=0;i<6;i++) {
+      int lastPatNum=info.ds->subsong[0]->orders.ord[i][info.ds->subsong[0]->ordersLen - 1];
+      DivPattern* newPat=info.ds->subsong[0]->pat[i].getPattern(info.maxPat,true);
+      DivPattern* lastPat=info.ds->subsong[0]->pat[i].data[lastPatNum];
```

Comment:
If lastPat is NULL, this is going to crash. Change to getPattern(lastPatNum,false).

(`false` in getPattern will not allocate a new pattern on failure, but instead return an "empty" pattern)